packages/console/src/components/Entities/generators.ts

```diff
@@ -16,56 +16,55 @@ export const executionFilterGenerator: {
   ) => FilterOperation[];
 } = {
   [ResourceType.DATASET]: noFilters,
-  [ResourceType.LAUNCH_PLAN]: ({ name }, version) =>
-    version
+  [ResourceType.LAUNCH_PLAN]: ({ name }, version) => [
+    {
+      key: 'launch_plan.name',
+      operation: FilterOperationName.EQ,
+      value: name,
+    },
+    ...(version
```

Comment:
[nit] we can actually also use && to make it more concise

...(version && [...])

Reply:
i did it that way because there's a difference between
`[1,2, ...[]]` and `[1,2,...undefined]`